backend/src/middlewares/healthCheck.middleware.ts

```diff
@@ -0,0 +1,17 @@
+import { Request, Response, NextFunction } from 'express'
+// Function to respond to health checks
+const healthCheckmiddleware = (
+  request: Request,
+  response: Response,
+  next: NextFunction
+): Response | NextFunction | void => {
+  // Health check routes
+  if (request.url === '/ping' || request.url === '/health' || request.url === '/') {
```

Comment:
Why not let a router handle this (e.g. `app.get(['/ping', '/health', '/'], healthCheckMiddleware)`) instead of checking manually?